commit ee916fd0fdb8f43dacaab431de3e1f7225039d72
Author: Wei Yongjun <yjwei@cn.fujitsu.com>
Date:   Sun Apr 17 17:28:01 2011 +0000

    sctp: change auth event type name to SCTP_AUTHENTICATION_EVENT
    
    This patch change the auth event type name to SCTP_AUTHENTICATION_EVENT,
    which is based on API extension compliance.
    
    Signed-off-by: Wei Yongjun <yjwei@cn.fujitsu.com>
    Signed-off-by: Vlad Yasevich <vladislav.yasevich@hp.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>

diff --git a/include/net/sctp/user.h b/include/net/sctp/user.h
index 793617ecbfdf..4525d8c7f71a 100644
--- a/include/net/sctp/user.h
+++ b/include/net/sctp/user.h
@@ -408,7 +408,8 @@ enum sctp_sn_type {
 	SCTP_SHUTDOWN_EVENT,
 	SCTP_PARTIAL_DELIVERY_EVENT,
 	SCTP_ADAPTATION_INDICATION,
-	SCTP_AUTHENTICATION_INDICATION,
+	SCTP_AUTHENTICATION_EVENT,
+#define SCTP_AUTHENTICATION_INDICATION	SCTP_AUTHENTICATION_EVENT
 };
 
 /* Notification error codes used to fill up the error fields in some